### PR TITLE
Adds raw TC to uplink and 10mm ammo variants

### PR DESCRIPTION
**Pull request by ghost**
Adds raw telecrystals to the uplink, can be used to refill uplinks and uplink implants, useful for TC trades. Also adds 10mm ammo variants to the uplink for traitors and nuke ops: Armor Piercing, Hollow Point (better against non-armor, worse against armor), and Incendiary. Incendiary does regular 30 brute and 2 shots on someone will ignite them.

Credit to @Fox-McCloud for the raw TC code.

:cl:
add: Added raw telecrystals to the uplink, can be used with uplinks and uplink implants.
add: Added 10mm ammo variants to uplink
/:cl:

### CODE REVIEW

**Comment by kevinz000** [comment]
oh i can't print tcs out from R&D im' sad now :(

**Comment by Mysak0CZ** [comment]
You use one TC at a time, why not entire stack?

**Comment by coiax** [comment]
Making TCs a stackable item would make sense.

Also, you should make them sellable in Cargo, I'm sure Nanotrasen would pay through the nose to obtain these.

**Comment by XDTM** [comment]
What if we had a NT luxury TC market that traitors could steal from to get extra TCs

File: code/game/objects/items/stacks/telecrystal.dm [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	flags = NOBLUDGEON
+	origin_tech = "materials=6;syndicate=1"
+
+/obj/item/stack/telecrystal/attack(mob/target, mob/user)
+	if(target == user) //You can't go around smacking people with crystals to find out if they have an uplink or not.
+		for(var/obj/item/weapon/implant/uplink/I in target)
+			if(I && I.implanted)
+				I.hidden_uplink.telecrystals += 1
+				use(1)
+				user << "<span class='notice'>You press the [src] onto yourself and charge your hidden uplink.</span>"
```
Comment (Fox-McCloud) [inline code/game/objects/items/stacks/telecrystal.dm]: `you press `src``

**Comment by Fox-McCloud** [comment]
Oh. Interesting to see these show up here. I added these so traitor could collaborate by smuggling these around or giving out a few extra points to each other, but one party wasn't sure what they wanted at the time.

**Comment by uraniummeltdown** [comment]
@coiax If I make them redeemable for Cargo points people will just buy an emag and empty out the uplink, then sell the crystals and buy tons of null crates

@Mysak0CZ It's just easier to do it one at a time, especially for nuke ops in the field so people don't have to keep buying more TC to redistribute if they get too many/few

**Comment by coiax** [comment]
Well, there already exists a TC->Cargo Credits conversion with buying suitcases full of Space Cash. I think you can get a pretty high amount from that.

**Comment by coiax** [comment]
I just did the numbers, assuming you get a 6 TC emag, you can afford 14 TC of Briefcases of Cash with 5000 each, making it 70000 cargo points, which is worth 5 NULL_ENTRY crates. The problem you describe already exists.

**Comment by Kierany9** [comment]
>more ammo variants

stop this meme these aren't useful

**Comment by KorPhaeron** [comment]
bloat

**Comment by optimumtact** [comment]
it's a bit late to complain now you dip

**Pull request merged**